### PR TITLE
[P4-636] Display cancellation banner on move detail page

### PR DESCRIPTION
**Pull request by teneightfive**
If a move has been cancelled it will display a banner to signify
that this move is in a cancelled state.

The banner can be used in the future to show the status of a move and any further
information like reason for cancellation.

🚀 **[DEMO](https://book-a-secure-move.herokuapp.com/)** 🚀 

## What it looks like

localhost_3000_move_a53fb8f7-3e34-41ca-8f7d-2b97ae89f785

### CODE REVIEW

File: app/move/controllers/view.js [hunk 1]
```diff
@@ -2,7 +2,11 @@ const presenters = require('../../../common/presenters')
 
 module.exports = function view(req, res) {
   const { move } = res.locals
-  const { person } = move
+  const {
+    person,
+    cancellation_reason: reason,
```
Comment (feedmypixel) [inline app/move/controllers/view.js]: could this benefit from being `cancelReason`?

File: app/move/controllers/view.js [hunk 2]
```diff
@@ -2,7 +2,11 @@ const presenters = require('../../../common/presenters')
...
+    cancellation_reason_comments: reasonComments,
```
Comment (feedmypixel) [inline app/move/controllers/view.js]: could this benefit from being `cancelReasonComments`?

File: common/components/message/_message.scss [hunk 3]
```diff
@@ -87,3 +93,22 @@
   border-width: 0 0 0 5px;
   background-color: govuk-colour("light-grey");
 }
+
+.app-message--temporary {
+  background-color: govuk-colour("blue");
+  border-width: 0;
+  color: govuk-colour("white");
+  padding: $_message-padding + 5;
```
Comment (feedmypixel) [inline common/components/message/_message.scss]: should we be aligning white space and not using magic numbers?

Comment (teneightfive) [reply common/components/message/_message.scss]: Do you mean the variable or the `5`? The 5 is the border width. 

Maybe that could be a variable so it's easier to understand.

Comment (feedmypixel) [reply common/components/message/_message.scss]: > Maybe that could be a variable so it's easier to understand.

This sounds good

**Pull request merged**